processing/src/main/java/io/druid/segment/column/SimpleDictionaryEncodedColumn.java

```diff
@@ -71,7 +72,7 @@ public IndexedInts getMultiValueRow(int rowNum)
   @Override
   public String lookupName(int id)
   {
-    return cachedLookups.get(id);
+    return Strings.emptyToNull(cachedLookups.get(id));
```

Comment:
I'm surprised we haven't encountered this one before, do you mind adding a comment to clarify which edge case this is handling, and maybe refer to the correponding test?